src/data/light.ts

```diff
@@ -56,7 +56,13 @@ export const lightSupportsDimming = (entity: LightEntity) => {
   );
 };
 
-export const getLightRgbColor = (entity: LightEntity): number[] | undefined =>
+export const getLightRgbColor = (
+  entity: LightEntity
+): [number, number, number] | undefined => entity.attributes.rgb_color;
+
+export const getLightRawRgbColor = (
```

Comment:
What's raw about this? Maybe name it `getLightCurrentModeRgbColor` ?

Reply:
We have rgb, that is a representative for the current color, and we have the raw rgb color, without the white channel values

Reply:
I am not completely following. You're saying that `entity.attributes.rgbww_color` does not contain a white value ?

Reply:
Yes it does, but the rgb of rgbww_color is not the same as rgb_color as that last one has the ww mixed in the rgb